parsl/executors/high_throughput/executor.py

```diff
@@ -479,38 +506,60 @@ def scale_out(self, blocks=1):
         r = []
         for i in range(blocks):
             if self.provider:
-                block = self.provider.submit(self.launch_cmd, 1, 1)
-                logger.debug("Launched block {}:{}".format(i, block))
-                if not block:
+                external_block_id = len(self.blocks)
```

Comment:
`str`ingify the external_block_id here so that it's already the right type elsewhere in the code

Reply:
Done